client/app/reducers/settings.coffee

```diff
@@ -0,0 +1,15 @@
+Immutable = require 'immutable'
+
+{ActionTypes} = require '../constants/app_constants'
+
+DEFAULTSTATE =
+    settings: Immutable.Map()
```

Comment:
I agree with you, let's keep the state as simple as possible. (no settings.settings)